Round_1_query_2.sql

```diff
@@ -1,2 +1,2 @@
-select * from Listings
+select top 10 * from Listings
```

Comment:
why the `top 10`? Lets maybe try `LIMIT 10` at the end of the query for better visibility